.github/workflows/integration-tests.yaml

```diff
@@ -25,6 +25,7 @@ on:
 
 jobs:
   compatibility-tests:
+    if: false # turning off for 3.0 iteration
```

Comment:
I didn't delete these because potentially we'll want to use this structure for 3.x compatibility tests